flutter/lib/sentry_flutter.dart

```diff
@@ -1,3 +1,4 @@
+// ignore: invalid_export_of_internal_element
```

Comment:
Can we ignore the specific element instead of the whole file? its easier to find and fix later on.

Reply:
I'm not sure how, do we have an example elsewhere?

Reply:
nvm, makes sense, its the export file.